src/client/components/SentReviewsComponent/SentReviewsComponent.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './SentReviewsComponent.styles.css';
+import unknownUser from '../../assets/images/portraits/unknown-black.png';
+import { RatingStars } from '../RatingStars/RatingStars';
+import { ReviewPost } from '../ReviewPost/ReviewPost';
+
+export const SentReviewsComponent = ({ reviews }) => {
+  return (
+    <div className="collection-reviews">
+      {reviews.map((review) => (
```

Comment:
You can destructure if you want, i.e `map({id, cv, reviewer}) => ...`.

Reply:
sure